tests/Keboola/TableResultFormatTest.php

```diff
@@ -46,6 +46,47 @@ public function testTableFormat(): void
             ],
         ], $table->getOutput());
     }
+    public function testDuplicatedTableColumn(): void
+    {
+        $table = new Table();
+        $table
+            ->setName('testName')
+            ->setCatalog('catalog')
+            ->setSchema('schema')
+            ->setType('view')
+            ->setRowCount(22);
+
+        $column = new TableColumn();
+        $column
+            ->setName('Asdno osdn')
+            ->setType('int');
+
+        $table->addColumn($column);
+
+        $column = new TableColumn();
+        $column
+            ->setName('Asdno osdn')
+            ->setType('varchar');
```

Comment:
A couple of things here.  1) we should be testing by ordinal position rather than name.  And 2), the second object that comes with the same ordinal position should have it's extra properties added to the first.  For example if the second object is an FK then it will have set `fk_name` and that should be present in the result.  Does that make sense?

Reply:
you're right